scripts/exhafs_analysis.sh

```diff
@@ -738,6 +754,10 @@ EOFdiag
    fi
 fi # End diagnostic file generation block - if [ $GENDIAG = "YES" ]
 
+# Save satbias data for next cycle
+${NCP} satbias_out  $RESTARTanl/satbias_hafs_out
+${NCP} satbias_pc.out  $RESTARTanl/satbias_hafs_pc.out
+
```

Comment:
Shouldn't this code block be protected by an if statement for ${ONLINE_SATBIAS} = "YES" ]? Otherwise, won't it rename the GDAS files as HAFS files even if online satellite bias correction hasn't been used?